test-resources/swift/cleanup_rules/input/SampleClass.swift

```diff
@@ -15,4 +15,6 @@ class SampleClass {
     var isEnabled3 = TestEnum.stale_flag_one.isEnabled && v1
     var isEnabled4 = f2() && TestEnum.stale_flag_one.isEnabled 
     var isEnabled5 = v2 && TestEnum.stale_flag_one.isEnabled 
+
```

Comment:
Let's add a pre-commit hook for swift formatting ? https://github.com/nicklockwood/SwiftFormat